### PR TITLE
Minor Edits on File>Open

### LINKED ISSUE #5261: Minor edit on File > Open
image

Here is what I would like on this dialogue.
Currently the file name at the top is left justified, so you have no idea of the name of the file.  The field is also very small. 
If it could be right-justified, as above, then you would know the name of the file you are opening.

### PR DESCRIPTION
**Pull request by Ivanluv**
fixes
#5261 
@africanmathsinitiative/developers  this is ready for review

### CODE REVIEW

File: instat/dlgImportDataset.vb [hunk 1]
```diff
@@ -48,6 +48,7 @@ Public Class dlgImportDataset
     Private bSupressCheckAllSheets As Boolean = False
     Private bSupressSheetChange As Boolean = False
 
+
```
Comment (Muthenya) [inline instat/dlgImportDataset.vb]: These spaces can be removed. No change was necessary on this file.

File: instat/static/InstatObject/R/instat_object_R6.R [hunk 2]
```diff
@@ -894,10 +894,11 @@ DataBook$set("public", "sort_dataframe", function(data_name, col_names = c(), de
 DataBook$set("public", "rename_dataframe", function(data_name, new_value = "", label = "") {
   data_obj <- self$get_data_objects(data_name)
   if(data_name != new_value) {
-    if(new_value %in% names(private$.data_sheets)) stop("Cannot rename data frame since ", new_value, " is an existing data frame.")
+   if(new_value == names(private$.data_sheets)) stop("Cannot rename data frame since ", new_value, " is an existing data frame.")
```
Comment (Muthenya) [inline instat/static/InstatObject/R/instat_object_R6.R]: This change does not answer anything from the issue neither do I think it answers a separate issue? @Ivanluv ?

**Comment by Muthenya** [comment]
@Ivanluv?

**Comment by maxwellfundi** [comment]
@Ivanluv It seems that there are unwanted files here that do not relate to the issue #5261. Please close this and transfer the changes on the Import dataset on a new branch and open a new PR

**Comment by Ivanluv** [comment]
Doing it now

**Pull request closed without merging**